docs/api/wanchain.md

```diff
@@ -2056,16 +2103,16 @@ This order kind is used for delegating `AssetLedger` deploy.
 |-|-
 | assetLedgerData.capabilities | [required] A list of `integers` representing ledger capabilities.
 | assetLedgerData.name | [required] A `string` representing asset ledger name.
-| assetLedgerData.owner | [required] A `string` representing Ethereum wallet, which will be the owner of the asset ledger.
+| assetLedgerData.owner | [required] A `string` representing Wanchain wallet, which will be the owner of the asset ledger.
 | assetLedgerData.schemaId | [required] A `string` representing data schema ID.
 | assetLedgerData.symbol | [required] A `string` representing asset ledger symbol.
 | assetLedgerData.uriPrefix | [required] A `string` representing prefix of asset URI.
 | assetLedgerData.uriPostfix | [required] A `string` representing postfix of asset URI.
 | expiration | [required] An `integer` number representing the timestamp in milliseconds after which the order expires and can not be performed any more.
 | kind | [required] An `integer` number that equals to `OrderKind.ASSET_LEDGER_DEPLOY_ORDER`.
-| makerId | [required] A `string` representing an Ethereum account address which makes the order. It defaults to the `accountId` of a provider.
+| makerId | [required] A `string` representing an Wanchain account address which makes the order. It defaults to the `accountId` of a provider.
```

Comment:
| makerId | [required] A `string` representing a Wanchain account address which makes the order. It defaults to the `accountId` of a provider.